platform/podjs/src/browserPod.ts

```diff
@@ -362,13 +362,16 @@ class PodJsInfo implements Info {
     }
 }
 
+/**
+ * @interface NetworkResponse
```

Comment:
`typedoc` [apparently does not support this](https://github.com/TypeStrong/typedoc/issues/1214#issuecomment-590004944)